COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/Point.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace COORDINATE_SYSTEM2
+{
+    internal class Point
+    {
+        private int x;
+        private int y;
+        public Point(int X, int Y)
```

Comment:
pls use Camel case for method parameters names

Reply:
done